dashboard/config/marketing/announcements.json

```diff
@@ -20,24 +20,8 @@
       "buttonUrl": "https://code.org/hourofcode",
       "buttonId": "hoc-2023"
     },
-    "hoc-2023-dance-ai": {
-      "image": "/images/dance-hoc/dance-party-activity-ai-edition.png",
-      "title": "Dance Party: AI Edition",
-      "body": "Learn about artificial intelligence (AI) concepts to create your own virtual dance party showcasing today's top artists. With dozens of songs to choose from, reach every student no matter their music taste. It's time to strut your stuff!",
-      "buttonText": "Learn more",
-      "buttonUrl": "https://code.org/dance",
-      "buttonId": "hoc-2023-dance-ai"
-    },
-    "ai-teaching-assistant": {
-      "dcdo": "ai-teaching-assistant-launch",
-      "image": "/images/action-blocks/action-block-ai-ta.png",
-      "title": "AI Teaching Assistant",
-      "body": "Elevate teaching confidence and streamline assessment of student progress. This tool heralds a new chapter in personalized learning and instructional excellence.",
-      "buttonText": "Learn more about AI Teaching Assistant",
-      "buttonUrl": "https://code.org/ai-ta",
-      "buttonId": "ai-teaching-assistant"
-    },
     "music-lab-launch": {
+      "dcdo": "music-lab-launch-2024",
```

Comment:
I do wonder what happens if multiple banners are supposed to be shown 🤔

Reply:
Since this banner is used for marketing purposes, it makes sense to only show one at a time, so this may be more of a timing problem. I don't think we can put multiples banners up, even if they can be displayed with DCDO flags.

Reply:
oh sorry, I was curious what actually happens if multiple of these DCDO flags are set. Totally agree that only one should show at one time.

Really, I'm asking if there's some potential work here to make sure we don't accidentally launch multiple banners at the same time (wayyyyy out of scope of this PR)

Reply:
Oh! To get a banner to show you have to update the `pages` object (see code below), which only takes one option at a time per page. Hypothetically it should be impossible to show more than one. 

I just tried setting both the `ai-teaching-assistant-launch` and `music-lab-launch-2024` DCDO flags to true locally, and only the Music Lab banner is showing up since that's what's set on the `pages` object. 

The only time I could see us _wanting_ to set multiples and controlling them with DCDO flags is if there is a tight launch turnaround and we want to turn one off (`ai-teaching-assistant-launch` for example) and one on (`music-lab-launch-2024`) without having to manually update this file. Like if we wanted to keep the AI TA banner up right up until Music Lab launched.

https://github.com/code-dot-org/code-dot-org/blob/7c1b5bee73ca9ec2992642b3fef2b825f8b5ce33/dashboard/config/marketing/announcements.json#L3-L4

Reply:
Got it thank you! Seems like it's working pretty well :)